dockstore-webservice/src/main/java/io/dockstore/webservice/resources/EventResource.java

```diff
@@ -72,7 +74,7 @@
 public class EventResource {
     private static final String PAGINATION_DEFAULT_STRING = "10";
     private static final String SUMMARY = "Get events based on filters.";
-    private static final String DESCRIPTION = "Optional authentication.";
+    private static final String DESCRIPTION = "Requires authentication.";
```

Comment:
I think changing this is changing the description for  `/events` as well which doesn't seem quite right.

Reply:
I changed this from "Optional" to "Requires" because the `/events` endpoint seemed to require authentication and the optional description was misleading. Am I missing something?

Reply:
Oh, I see. `EventSearchType` doesn't have any enum types that apply without a specific user. 
Ok, I think that's ok then